client/src/components/Home.js

```diff
@@ -2,15 +2,23 @@ import React from 'react';
 
 class Home extends React.Component {
 
+    handleNameChange = (event) => {
+        this.props.updateTeam({ name: event.target.value});
+    }
+
+    createTeam = () => {
+        this.props.registerTeam(this.props.name);
+    }
+
     render () {
         return (
             <div className="Home">
                 <h1>Create Team</h1>
                 <label>
                     Team Name: 
-                    <input type='text'/>
+                    <input type='text' onChange={this.handleNameChange} />
```

Comment:
Do you need to bind this to handleNameChange handler?

Reply:
No need to bind because of the way the method is declared. this is automatically bound to it because it is using an arrow function